bin/github-backup

```diff
@@ -780,17 +787,26 @@ def fetch_repository(name,
             git_command = ['git', 'remote', 'set-url', 'origin', remote_url]
             logging_subprocess(git_command, None, cwd=local_dir)
 
-        git_command = ['git', 'fetch', '--all', '--force', '--tags', '--prune']
+        if lfs_clone:
+            git_command = ['git', 'lfs', 'fetch', '--all', '--force', '--tags', '--prune']
```

Comment:
Does this also get the non-lfs content?

Reply:
Yes, it does.

You do need to have ```git-lfs``` installed though.

https://git-lfs.github.com

Reply:
We should both document that and also toss in a check to ensure its installed if the `lfs` flag is used.